package.json

```diff
@@ -150,7 +152,8 @@
     "zone.js": "^0.8.4"
   },
   "dependencies": {
-    "@swimlane/ngx-charts": "^7.0.1",
+    "@angular/cdk": "^6.0.1",
```

Comment:
let's make this a peerDependency please.

Reply:
👍 Good catch